maker/src/main.rs

```diff
@@ -44,11 +46,23 @@ async fn main() -> Result<()> {
         tracing::info!(address, "New address");
 
         loop {
-            match wallet::get_balance() {
-                Ok(balance) => tracing::info!(?balance, "Current balance"),
-                Err(e) => tracing::error!("Could not retrieve balance: {e:#}"),
+            let started = Instant::now();
+            let wallet_result = wallet::get_balance();
+            let duration = started.elapsed();
+            let sync_time_in_seconds = duration.as_secs();
+            match wallet_result {
+                Ok(balance) => tracing::info!(?balance, sync_time_in_seconds, "Current balance"),
+                Err(e) => {
+                    tracing::error!(sync_time_in_seconds, "Could not retrieve balance: {e:#}")
+                }
             }
-            tokio::time::sleep(Duration::from_secs(10)).await;
+            let sync_time = match network {
```

Comment:
Am I understanding this correct that the  `maker` has a less frequent sync interval than the `taker`? 
I wonder if that can cause errors where the taker has already caught up with a state on chain, whereas the maker did not and errors.

Reply:
It's time to find out 😅.